.github/workflows/release.yml

```diff
@@ -392,6 +395,15 @@ jobs:
                   cd -
               fi
           done
+          for d in ci/tests/*/
```

Comment:
`ci/tests/` to run before the repo specific ones as a failure there would not run these tests.